api/nameguard/nameguard.py

```diff
@@ -350,6 +351,14 @@ async def secure_primary_name(self, address: str, network_name: str) -> SecurePr
                                        primary_name_status=status,
                                        nameguard_result=nameguard_result)
 
+    async def fake_eth_name_check_fields(self, network_name, contract_address, token_id, investigated_fields: dict[str,str]) -> FakeEthNameCheckResult:
+        contract_address = contract_address.lower()
+        
+        # title = investigated_fields['title']  # mandatory if ENS contract; ENS name
```

Comment:
@lightwalker-eth In the new endpoint a client needs to provide ETH name under `title` key of `investigated_fields` dictionary if contract_address is ENS, because we return NameGuard report.

Reply:
@lightwalker-eth It generates a problem when we want to return fields which are faking ENS. If user also send `fields` dict with `title` key.